compiler.rst

```diff
@@ -394,7 +394,10 @@ Changing this number will lead to all .pyc files with the old ``MAGIC_NUMBER``
 to be recompiled by the interpreter on import.  Whenever ``MAGIC_NUMBER`` is
 changed, the ranges in the ``magic_values`` array in :file:`PC/launcher.c`
 must also be updated.  Changes to :file:`Lib/importlib/_bootstrap_external.py`
-will take effect only after running ``make regen-importlib``.
+will take effect only after running ``make regen-importlib``. Running this 
+command before adding the new bytecode to :file:`Python/ceval.c` will result 
+in an error. You should proceed to the next step before importlib is 
```

Comment:
~~You should ... regenerated~~

Instead, you should add the new bytecode and then run `make regen-importlib`.

Reply:
Fair point. What would you think of:
"Running this command before adding the new bytecode target to :file:`Python/ceval.c` will result in an error. You should only run ``make regen-importlib`` after the new bytecode target has been added."

Or maybe we can remove the second sentence. I was referring to the next step as that one is the one explaining how to deal with the ceval.c file. Thanks for your input!

Reply:
Fair point. What would you think of:
"Running this command before adding the new bytecode target to :file:Python/ceval.c will result in an error. You should only run make regen-importlib after the new bytecode target has been added."

Or maybe we can remove the second sentence. I was referring to the next step as that one is the one explaining how to deal with the ceval.c file. Thanks for your input!

Reply:
Your suggested wording works for me @psyker156 Thanks 👍

Reply:
Alright! I will patch it up! Thanks a lot!